include/glow/Backends/BackendUtils.h

```diff
@@ -81,7 +81,24 @@ class RuntimeBundle {
   /// Sets the input and output flags for each symbol in the symbolBundle.
   void setInputsandOutputs();
 
-  RuntimeBundle() = default;
+  /// Computes offsets and total allocation for Constants, Placeholders, and
+  /// Activations to build runtime symbol table. Returns RuntimeBundle.
+  static runtime::RuntimeBundle create(const IRFunction &F,
+                                       MemoryAllocator &constantAllocator,
+                                       MemoryAllocator &placeholderAllocator,
+                                       MemoryAllocator &activationsAllocator);
+
+  /// Build a runtime symbol table from a Function.  Computes Constant and
+  /// Placeholder sizes, but not Activations, since Functions are unserialized.
+  /// Only use this method to generate bundles for backends that do not use
+  /// Glow's IR.
+  static runtime::RuntimeBundle create(const Function *F);
```

Comment:
nit: why not ref here? (consistent with the other `create` method, does not allow to pass nullptr)

Reply:
Sure, I can do that.  We use `Function *` way more often than `Function &` but I can't see any good reason for it...